sass/responsive/_mobile.scss

```diff
@@ -1370,7 +1370,7 @@
     .search-bar__container {
         @include flex(0 0 50px);
         background: $primary-color;
-        color: $white;
+        color: var(--sidebar-header-text-color);
```

Comment:
@jgilliam17, this is supposed to set the text colour for the search bar header in mobile, but unfortunately the input field doesn't inherit it and so is not being styled properly. We have a separate ticket to handle that.